localstack-core/localstack/services/lambda_/provider.py

```diff
@@ -1314,7 +1314,7 @@ def delete_function(
             # delete a version of the function
             version = function.versions.pop(qualifier, None)
             if version:
-                self.lambda_service.stop_version(version.qualified_arn())
+                self.lambda_service.stop_version(version.id.qualified_arn())
```

Comment:
maybe I'm missing something but the version is defined as
```
@dataclasses.dataclass(frozen=True)
class FunctionVersion:
    id: VersionIdentifier
    config: VersionFunctionConfiguration

    @property
    def qualified_arn(self) -> str:
        return self.id.qualified_arn()
```

so why would this change the behavior :thinking:

Reply:
ah ok, just saw Joel's comment, makes sense. Kinda ignored the additional `()` there!